### ISSUE TITLE
datacarpentry/astronomy-python

### ISSUE THREAD

**Issue by zkamvar** [carpentries/lesson-transition#89]
tracking issue for https://github.com/datacarpentry/astronomy-python/

 - [x] figure links are not properly linked. Part of the reason for this is because the image links are coded as HTML images, not markdown images 😩 (see https://github.com/carpentries/pegboard/issues/126 for why this is awkward)
 - [x] Fix filter _However_ some of the figures are not being ported over and ~~I need to _figure_ out why.~~ OH. IT'S BECAUSE THE DURN FILTER IS CATCHING THEM. (fixed in 0e63cebd468d97eacf9e81ae43b32bfdd64c758e)
 - [x] one root link is missing in instructor notes
 - [x] packs too big, need to increase http.postBuffer size (fixed in  c182807)

```markdown
episodes/02-coords.md:307 [missing file]: [](fig/gd1-4.png)
episodes/04-motion.md:152 [missing file]: [](../fig/gd1-1.png)
episodes/04-motion.md:223 [missing file]: [](fig/gd1-4.png)
episodes/04-motion.md:398 [missing file]: [](../fig/gd1-1.png)
episodes/05-select.md:548 [missing file]: [](../fig/gd1-2.png)
episodes/05-select.md:558 [missing file]: [](../fig/gd1-4.png)
episodes/06-join.md:29 [missing file]: [](../fig/gd1-3.png)
episodes/06-join.md:413 [missing file]: [](../fig/join.png)
episodes/07-photo.md:31 [missing file]: [](../fig/gd1-3.png)
episodes/07-photo.md:107 [missing file]: [](../fig/gd1-3.png)
episodes/08-plot.md:124 [missing file]: [](../fig/gd1-5.png)
instructors/instructor-notes.md:219 [missing file]: [Making the Isochrone Dataframe](/astronomy-python/calculating_MIST_isochrone)
```

**Comment by zkamvar** [comment]
fun git error when pushing to the test repo:

```
from datacarpentry/astronomy-python to fishtree-attempt/astronomy-python (bots)
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  7896  100  7802  100    94   4931     59  0:00:01  0:00:01 --:--:--  4991
Enumerating objects: 1605, done.
Counting objects: 100% (1605/1605), done.
Delta compression using up to 8 threads
Compressing objects: 100% (557/557), done.
error: unable to rewind rpc post data - try increasing http.postBuffer
error: RPC failed; curl 56 GnuTLS recv error (-12): A TLS fatal alert has been received.
send-pack: unexpected disconnect while reading sideband packet
Writing objects: 100% (1605/1605), 60.25 MiB | 66.91 MiB/s, done.
Total 1605 (delta 1063), reused 1541 (delta 1036), pack-reused 0
fatal: the remote end hung up unexpectedly
Everything up-to-date
make: *** [Makefile:130: sandpaper/datacarpentry/astronomy-python-status.json] Error 1
```

**Comment by zkamvar** [comment]
Actually.... I just got the git issue again. I don't think this is transient. This lesson has a LOT of large files:

```
$ dust astronomy-python/
 2.1M     ┌── notebooks                                             │               █ │   2%
 2.2M   ┌─┴ _extras                                                 │               █ │   2%
 1.3M   │ ┌── 05-select_files                                       │              ░█ │   1%
 1.4M   │ ├── 04-motion_files                                       │              ░█ │   1%
 2.5M   │ ├── 08-plot_files                                         │              ░█ │   2%
 9.2M   ├─┴ fig                                                     │              ██ │   8%
 6.4M   │   ┌── gd1_results.fits                                    │             ▒▒█ │   5%
  14M   │   ├── gd1_data.hdf                                        │             ▒██ │  12%
  21M   │ ┌─┴ data                                                  │             ███ │  18%
  21M   ├─┴ student_download                                        │             ███ │  18%
 1.4M   │ ┌── gd1_df5.hdf5                                          │           ░░░░█ │   1%
 1.4M   │ ├── gd1_candidates.hdf5                                   │           ░░░░█ │   1%
 1.8M   │ ├── gd1_merged.hdf5                                       │           ░░░░█ │   2%
 8.6M   │ ├── gd1_results.fits                                      │           ░░░██ │   7%
  16M   │ ├── gd1_dataframe.hdf5                                    │           ░░███ │  14%
  29M   ├─┴ data                                                    │           █████ │  25%
  54M   │     ┌── pack-434523be16b9e6fe8054d2c2e18977d6df4f45f7.pack│        ████████ │  46%
  54M   │   ┌─┴ pack                                                │        ████████ │  46%
  54M   │ ┌─┴ objects                                               │        ████████ │  46%
  55M   ├─┴ .git                                                    │        ████████ │  47%
 118M ┌─┴ astronomy-python                                          │████████████████ │ 100%
```

With the 54 MB

**Comment by zkamvar** [comment]
Note, I was able to get this now, but I think the errors above are still prudent:

```
from datacarpentry/astronomy-python to fishtree-attempt/astronomy-python (bots)
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  7896  100  7802  100    94   4242     51  0:00:01  0:00:01 --:--:--  4291
Enumerating objects: 1605, done.
Counting objects: 100% (1605/1605), done.
Delta compression using up to 8 threads
Compressing objects: 100% (557/557), done.
Writing objects: 100% (1605/1605), 60.25 MiB | 9.98 MiB/s, done.
Total 1605 (delta 1063), reused 1541 (delta 1036), pack-reused 0
remote: Resolving deltas: 100% (1063/1063), done.
To https://github.com/fishtree-attempt/astronomy-python.git
 * [new branch]      main -> main
branch 'main' set up to track 'origin/main'.
✔ Setting active project to '/home/zhian/Documents/Carpentries/Git/carpentries/lesson-transition/sandpaper/datacarpentry/astronomy-python'
✔ Initializing empty, orphan 'gh-pages' branch in GitHub repo 'fishtree-attempt/astronomy-python'
✔ GitHub Pages is publishing from:
• URL: 'https://fishtree-attempt.github.io/astronomy-python/'
• Branch: 'gh-pages'
• Path: '/'
Browse the repository at https://github.com/fishtree-attempt/astronomy-python
```

**Comment by zkamvar** [comment]
I believe I have addressed the issue by increasing the size of the http.postBuffer in the local config for this repo.

I followed https://medium.com/swlh/everything-you-need-to-know-to-resolve-the-git-push-rpc-error-1a865fd1ebea for the solution.

**Comment by zkamvar** [comment]
This lesson will be converted to use [The Carpentries Workbench][workbench]
To prevent accidental reversion of the changes, we are temporarily revoking
write access for all collaborators on this lesson:

 - [ ] @rkotulla (maintain)
 - [ ] @AllenDowney (maintain)
 - [ ] @philrosenfield (maintain)
 - [ ] @abostroem (maintain)
 - [ ] @bmorris3 (push)
 - [ ] @ivastar (push)
 - [ ] @fmichonneau (maintain)
 - [ ] @mrawls (push)
 - [ ] @tautme (push)
 - [ ] @rudyphd (maintain)
 - [ ] @HeatherAn (push)
 - [ ] @rlmcclure (push)

If you no longer wish to have write access to this repository, you do not
need to do anything further.

1. What you can expect from the transition 📹: https://carpentries.github.io/workbench/beta-phase.html#beta
2. How to update your local clone 💻: https://carpentries.github.io/workbench/beta-phase.html#updating-clone
3. How to update (delete) your fork (if you have one) 📹: https://carpentries.github.io/workbench/faq.html#update-fork-from-styles

If you wish to regain write access, please re-clone the repository on your machine and
then comment here with `I am ready for write access :rocket:` and the
admin maintainer of this repository will restore your permissions.

If you have any questions, please reply here and tag @zkamvar

[workbench]: https://carpentries.github.io/workbench

**Comment by zkamvar** [comment]
fixed in 98bba7891f513a68cdaaab574cf4ef3eb0c1dd2e
